### PR TITLE
Feature/clerks list

### PR DESCRIPTION
**Pull request by jake6642**
This includes the clerk list page, clerk update page, and reset password functionality. I had to add a getter/setter to the Company for a Map of users. It pulls in a fake list of users, displays them, and allows the user to update them or reset password. It is assumed that only the admin can navigate to this page because no checks for admin are in place

### CODE REVIEW

File: src/acme/ui/AcmeUI.java [hunk 1]
```diff
@@ -21,7 +21,10 @@ public AcmeUI() {
         setLocationRelativeTo(null);
         addWindowListener(new ShutdownListener());
 
-        reports();
+        //TODO implement shutdownlistener and remove this
+        setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
```
Comment (simpleman19) [inline src/acme/ui/AcmeUI.java]: Shutdown listener is implemented, look at line 22 addWindowListener

Comment (jake6642) [reply src/acme/ui/AcmeUI.java]: The problem is, the listener doesn't do anything yet. The process stays running without the default close operation here or the listener being completed.

Comment (simpleman19) [reply src/acme/ui/AcmeUI.java]: It is fixed on my other pull request, this will just cause merging issues later

File: src/acme/ui/AcmeUI.java [hunk 2]
```diff
@@ -21,7 +21,8 @@ public AcmeUI() {
         setLocationRelativeTo(null);
         addWindowListener(new ShutdownListener());
 
-        reports();
+        userList();
```
Comment (jake6642) [inline src/acme/ui/AcmeUI.java]: I left this is for the sake of the test. Let me know if I need to remove it

**Comment by simpleman19** [comment]
Also I vote we need an admin checkbox but I don't remember if this was in the design doc

Feel free to ignore my comments and finish the pull request everything else looks good

**Comment by simpleman19** [comment]
I already ran acceptance criteria

**Pull request merged**